apps/nrm/views/partials/pv-contact-details-email-input.html

```diff
@@ -1,4 +1,4 @@
-<fieldset id="pv-contact-details-email-input-fieldset" class="panel-indent">
+<fieldset id="pv-contact-details-email-input-fieldset" class="govuk-checkboxes__conditional" aria-hidden="false">
```

Comment:
Changed to a div, removed the aria-hidden property since it's automatically picked up, and added the conditional class. Applied the same to all the other partials below.